examples/with-apollo-typescript/pages/launches-client.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { MainLayout } from '@Components/MainLayout'
+import { Launches } from '@Components'
+
+const LaunchesClient = (): JSX.Element => {
```

Comment:
the `JSX.Element` type shouldn't be needed, we can rely on TS inferring our type in this case, or you could use `React.FC` which is more useful than `JSX.Element`.